### ISSUE TITLE
PodFromCoreObject required fix

### ISSUE THREAD

**Issue by adisos** [np-guard/netpol-analyzer#44]
`PodFromCoreObject` should not always issue an error / ignore pods not scheduled or not assigned IP addresses.
Only on live cluster mode this is relevant, as opposed to handling pod resources from dir.

**Comment by elevran** [comment]
a workaround (instead of passing the "live or file based mode") may be to assign a fake IP to all Pods read from files?

**Comment by adisos** [comment]
> @adisos - might a workaround (instead of passing the "live or file based mode" to `PodToCoreObject`) be to assign a fake IP to all Pods read from files? Can probably use setting up a network+netmask from [netip](https://pkg.go.dev/net/netip) and then calling `Next()` next on it to generate the addresses (careful no to overlap with CIDR's being queried - CGNAT or localhost ranges may be a reasonable base)

Right, sounds good. Will consider this option as well to handle this issue.

**Comment by shireenf-ibm** [comment]
A merged  [PR127](https://github.com/np-guard/netpol-analyzer/pull/127) resolves this issue (assigning a fake IP to Pods from files).
However, keeping this issue open in order to consider using the [netip](https://pkg.go.dev/net/netip) for generating the fake ips in the future.
